src/VirtualClient/VirtualClient.Actions/Furmark/FurmarkMetricsParser.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using VirtualClient.Common.Contracts;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    ///  Parser for Graph500 output document.
```

Comment:
Update the summary according to Furmark. Please check it throughout and update it accordingly.